pyrs/core/mantid_helper.py

```diff
@@ -31,18 +31,18 @@ def generate_mantid_workspace(hidra_workspace, workspace_name, mask_id=None):
         checkdatatypes.check_string_variable('Workspace name', workspace_name)
 
     # Get data from HiDRA Workspace
-    two_theta_matrix, data_y_matrix = hidra_workspace.get_reduced_diffraction_data_set(mask_id)
+    two_theta_matrix, data_y_matrix, data_e_matrix = hidra_workspace.get_reduced_diffraction_data_set(mask_id)
 
     # Mantid (2019.11) does not accept NaN
     # Convert all NaN to zero.  No good peak will have NaN or Zero
     data_y_matrix[np.where(np.isnan(data_y_matrix))] = 0.
-    data_e_matrix = np.sqrt(data_y_matrix)
+    # data_e_matrix = np.sqrt(data_y_matrix)
```

Comment:
You can just delete the line. Revision control will keep track of what things used to be.
```suggestion
```